### ISSUE TITLE
Allow Blueprints to be used for multiple Collections

### ISSUE THREAD

**Issue by patrickhoehn** [statamic/ideas#1102]
I have collections for different purposes but they share exactly the same fields. I think it would be nice to be able to use blueprints for multiple collections instead of packing the whole blueprint into a fieldset and importing this into other blueprints.

**Comment by 925dk** [comment]
I agree. As a rookie, this was my first Statamic wtf. Are there technical reasons for this restriction, or are you trying to hint some form of content structuring? It seems like a pseudo restriction.

**Comment by fpolli** [comment]
I echo this. It would be inordinately helpful for my use case
